python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -777,3 +785,329 @@ def unroll(self, length, inputs, begin_state=None, layout='NTC', merge_outputs=N
+                                          h2h_kernel=h2h_kernel, h2h_dilate=h2h_dilate,
+                                          i2h_kernel=i2h_kernel, i2h_stride=i2h_stride,
+                                          i2h_pad=i2h_pad, i2h_dilate=i2h_dilate,
+                                          i2h_weight_initializer=i2h_weight_initializer,
+                                          h2h_weight_initializer=h2h_weight_initializer,
+                                          i2h_bias_initializer=i2h_bias_initializer,
+                                          h2h_bias_initializer=h2h_bias_initializer,
+                                          activation=activation,
+                                          prefix=prefix, params=params)
+
+    def state_info(self, batch_size=0):
+        return [{'shape': (batch_size,)+self._state_shape, '__layout__': self._conv_layout}]
+
+    def _alias(self):
+        return 'conv_rnn'
+
+    @property
+    def _gate_names(self):
+        return ('',)
+
+    def hybrid_forward(self, F, inputs, states, i2h_weight,
+                       h2h_weight, i2h_bias, h2h_bias):
+        prefix = 't%d_'%self._counter
+        i2h, h2h = self._conv_forward(F, inputs, states,
+                                      i2h_weight, h2h_weight, i2h_bias, h2h_bias,
+                                      prefix)
+        output = self._get_activation(F, i2h + h2h, self.activation,
+                                      name=prefix+'out')
+        return output, [output]
+
+
+class ConvLSTMCell(BaseConvRNNCell):
+    """Convolutional LSTM network cell.
+
+    Reference:
+        Xingjian et al. NIPS2015
+
+    Parameters
+    ----------
+    hidden_size : int
+        Number of units in output symbol.
+    conv_layout : str, default 'NCHW'
+        Layout of ConvolutionOp
+    h2h_kernel : tuple of int, default (3, 3)
+        Kernel of Convolution operator in state-to-state transitions.
+    h2h_dilate : tuple of int, default (1, 1)
+        Dilation of Convolution operator in state-to-state transitions.
+    i2h_kernel : tuple of int, default (3, 3)
+        Kernel of Convolution operator in input-to-state transitions.
+    i2h_stride : tuple of int, default (1, 1)
+        Stride of Convolution operator in input-to-state transitions.
+    i2h_pad : tuple of int, default (1, 1)
+        Pad of Convolution operator in input-to-state transitions.
+    i2h_dilate : tuple of int, default (1, 1)
+        Dilation of Convolution operator in input-to-state transitions.
+    activation : str or Block
+        default LeakyReLU(alpha=0.2)
+        Type of activation function.
+    prefix : str, default 'ConvLSTM_'
+        Prefix for name of layers (and name of weight if params is None).
+    params : RNNParams, default None
+        Container for weight sharing between cells. Created if None.
+    forget_bias : bias added to forget gate, default 1.0.
+        Jozefowicz et al. 2015 recommends setting this to 1.0
```

Comment:
I find that the forget_bias is not used. Also, for ConvLSTM, there seems to be not much difference for different forget_bias. I think it's safe to simply remove this comment.